Python.gitignore

```diff
@@ -43,6 +43,7 @@ htmlcov/
 nosetests.xml
 coverage.xml
 *,cover
+.hypothesis/
```

Comment:
And you're right: it seems bug in my ide and it thinks that if path contains trailing slash so that rule shouldn't get applied to subdirectories. I should have to check git instead. Sorry for noise, everything is correct here!